src/components/task-step/ends.cjsx

```diff
@@ -60,6 +60,7 @@ HomeworkEnd = React.createClass
   onNextStep: ->
 
   renderReviewSteps: (steps, label, type) ->
+    TaskStepActions.HACK_DO_NOT_RELOAD(true)
```

Comment:
eep, I'm worried about including this in actual code; is there a better way to handle this?